### PR TITLE
Add sample document

### PR DESCRIPTION
**Pull request by garg-anuj**

### CODE REVIEW

File: sample.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# First title
```
Comment (ilvaman) [inline sample.md]: hiiii

Comment (karangupta-dev) [inline sample.md]: good

Comment (AmanGarg97) [inline sample.md]: Header?

Comment (g33kyshivam) [inline sample.md]: Awesome !

Comment (kartikchauhan) [inline sample.md]: Could you please explain why did you add this file?

Comment (rv1996) [inline sample.md]: why only one header is there?

Comment (garg-anuj) [reply sample.md]: yes it is

Comment (garg-anuj) [reply sample.md]: Yo bro

Comment (asheesh-kumar-sharma) [reply sample.md]: hi ...

Comment (asheesh-kumar-sharma) [reply sample.md]: hi..

Comment (ilvaman) [inline sample.md]: hii

Comment (Mohitphulera) [inline sample.md]: ok

File: sample.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# First title but refined
```
Comment (rv1996) [inline sample.md]: only one header?
